presto-session-property-managers/src/main/java/io/prestosql/plugin/session/file/FileSessionPropertyManager.java

```diff
@@ -94,6 +95,19 @@ public FileSessionPropertyManager(FileSessionPropertyManagerConfig config)
     @Override
     public Map<String, Map<String, String>> getCatalogSessionProperties(SessionConfigurationContext context)
     {
-        return ImmutableMap.of();
+        // later properties override earlier properties
```

Comment:
Can you please add test that covers that? I guess we need few files with catalog properties and see what get actually loaded.

Reply:
Testing jackson serialization/deserialization is handled in TestFileSessionPropertyManager.assertProperties
Simplified AbstractTestSessionPropertyManager.testMultipleMatch to only test merging
Moved the test of overrides into a dedicated AbstractTestSessionPropertyManager.testOverrides